Userland/Utilities/less.cpp

```diff
@@ -162,9 +162,7 @@ class Pager {
             write_range(m_line, m_subline, m_height - 1);
         }
 
-        status_line();
-
-        fflush(m_tty);
+        full_redraw();
```

Comment:
Ideally the commit message would tell us what exactly the bug was and how this fixes it. 

For example why do we need to do a full redraw all the time now? Is that really necessary? 

Also https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md also states: `Don't Use weasel-words like "refactor" or "fix" to avoid explaining what's being changed.

Reply:
Well I was trying out some stuff to try to fix the coloring but forgot to remove this change that is actually unneeded

Reply:
The commit message is now wrong since it doesn't touch `less`

Reply:
Oops, missed that comment, fixed